2017/AUG/AUG17/RAINBOWA/RAINBOWA.java

```diff
@@ -0,0 +1,55 @@
+//Made by rachanasoma
+import java.util.*;
+import java.lang.*;
+public class RAINBOWA {
+   public static void main(String args[]) {
+       Scanner sc = new Scanner(System.in);
+       int T, N, bef, a, flag, flag1;
```

Comment:
Please reduce the scope of the variables as much as possible.

Reply:
It's better to make `flag` and `flag1` as boolean type